options/simple_options.hpp

```diff
@@ -38,6 +38,8 @@ namespace bzn::option_names
     const std::string MEM_STORAGE = "mem_storage";
     const std::string MONITOR_ADDRESS = "monitor_address";
     const std::string MONITOR_PORT = "monitor_port";
+    const std::string MONITOR_COLLATE = "MONITOR_COLLATE";
```

Comment:
perhaps the text for 
	    const std::string MONITOR_COLLATE = "MONITOR_COLLATE";
	    const std::string MONITOR_COLLATE_INTERVAL_SECONDS = "MONITOR_COLLATE_INTERVAL_SECONDS";

should be lowercase.